### PR TITLE
Bump package versions to latest

### PR DESCRIPTION
**Pull request by nguerrera**
See individual commits for more details

### CODE REVIEW

**Comment by nguerrera** [comment]
cc @jhendrixMSFT @weshaggard 

Joel, I'm assuming that the infrastructure can bump BuildValues.props to 00013 and then upload both xunit and builtdools on myget after this is merged.

**Comment by nguerrera** [comment]
Hold off on merging this. I'm hitting test failures in corefx when bumping all packages to latest due to missing Internal.Uri.dll.

This just proves that it is imperative that we always run against latest. We should be updating packages all the time and all of this version bumping should be automated! We desperately need a -* wildcard nuget feature as in Project K.

**Comment by nguerrera** [comment]
cc @tarekgh @weshaggard @jhendrixMSFT

**Comment by nguerrera** [comment]
@dotnet-bot test this please
 
(It passed already, but doesn't seem to have used latest commits)

**Comment by nguerrera** [comment]
@dotnet-bot test this please

File: src/Microsoft.DotNet.Build.Tasks/Targets/sign.targets [hunk 1]
```diff
@@ -20,11 +20,17 @@
   </PropertyGroup>
 
   <!-- 
-    Compile gets hit directly by Intellisense builds in VS- while @(IntermediateAssembly) is set it won't actually exist.
-    PrepareForRun is the outer target in the core build depends chain that copies the intermediate assemblies to the output folder.
+    NOTE: This mechanism for wiring in the OpenSourceSign target can't be changed to any of the following:
+
+        * AfterTargets=Compile -> hit by intellisense builds while @(IntermediateAssembly) doesn't exist yet.
+        * AfterTargets=PrepareForRun -> hit after @(IntermediateAssembly) has already been copied to output.
+        * BeforeTargets=CopyFilesToOutputDirectory -> does not work on Mono.
```
Comment (nguerrera) [inline src/Microsoft.DotNet.Build.Tasks/Targets/sign.targets]: Hold off on merging, still need to test this.

**Comment by nguerrera** [comment]
Fixed the issues with Internal.Uri and more.

**Comment by jhendrixMSFT** [comment]
LGTM

**Pull request merged**